client/css/style.css

```diff
@@ -815,25 +815,30 @@ button {
 #chat .time,
 #chat .from,
 #chat .text {
-	display: table-cell;
+	display: block;
 	padding: 2px 0;
-	vertical-align: top;
+	flex: 0 0 auto;
 }
 
 #chat .time {
 	color: #ddd;
 	text-align: right;
-	max-width: 46px;
-	min-width: 46px;
+	width: 46px;
 }
 
 #chat .from {
 	border-right: 1px solid #f6f6f6;
 	color: #b1c3ce;
 	padding-right: 10px;
 	text-align: right;
-	max-width: 134px;
-	min-width: 134px;
+	width: 134px;
+	align-self: stretch;
```

Comment:
According to https://css-tricks.com/snippets/css/a-guide-to-flexbox/, `stretch` is the default value of `align-items`, and `align-self` is used to override it. Since this is not using `align-items`, there is nothing to override. Removing.